content/spin/v2/writing-apps.md

```diff
@@ -139,15 +139,22 @@ Multiple components can have the same source.  An example is a document archive,
 
 At the Wasm level, a Spin component is a Wasm component or module that exports a handler for the application trigger.  At the developer level, a Spin component is a library or program that implements your event handling logic, and uses Spin interfaces, libraries, or tools to associate that with the events handled by Spin.
 
-See the Language Guides section for how to do this in your preferred language. As an example, this is a component written in the Rust language. The `hello_world` function uses an attribute `#[http_component]` to identify the function as handling a Spin HTTP event. The function takes a `Request` and returns a `Result<Response>`:
+See the Language Guides section for how to do this in your preferred language. As an example, this is a component written in the Rust language. The `hello_world` function uses an attribute `#[http_component]`.
```

Comment:
Not sure why the explanation of the attribute and signature were removed here; without them this paragraph seems a bit bare.